packages/fx-core/resource/package.nls.json

```diff
@@ -853,7 +853,7 @@
   "depChecker.dotnetInstallStderr": "dotnet-install command failed without error exit code but with non-empty standard error.",
   "depChecker.dotnetInstallErrorCode": "dotnet-install command failed.",
   "depChecker.failToInstallDotnet": "Failed to install @NameVersion. Install @NameVersion manually and restart Visual Studio Code.",
-  "depChecker.NodeNotFound": "Cannot find Node.js.\n\nTeams Toolkit requires Node.js; the recommended version is v14.\n\nClick \"Learn more\" to learn how to install the Node.js.\n\n(If you just installed Node.js, restart Visual Studio Code for the change to take effect.)",
+  "depChecker.NodeNotFound": "Cannot find Node.js.\n\nTeams Toolkit requires Node.js; the recommended version is v16.\n\nClick \"Learn more\" to learn how to install the Node.js.\n\n(If you just installed Node.js, restart Visual Studio Code for the change to take effect.)",
```

Comment:
Why are some strings in the nls file (for localization) and some strings are not?

Reply:
Prerequiste check code in **packages/extension** and **packages/cli** not use nls file.
Env checker code in **packages/fx-core** was adapted by **huajie**